lib/Config.js

```diff
@@ -91,7 +91,7 @@ function locationConstraintAssert(locationConstraints) {
 function cosParse(chordCos) {
     // Cos number should only be first digit of config value
     const firstDigit = chordCos[0];
```

Comment:
can we remove this now and just do a pareInt on chordCos?  Is the former second digit referenced anywhere?